actionpack/lib/action_controller/metal/http_authentication.rb

```diff
@@ -106,11 +106,11 @@ def decode_credentials(request)
       end
 
       def auth_scheme(request)
-        request.authorization.split(' ', 2).first
+        (request.authorization || '').split(' ', 2).first
```

Comment:
Thanks.

Will it be a performance issue to evaluate ``request.authorization`` twice?
``` ruby
request.authorization.split(' ', 2).first if request.authorization
```